modules/tooling-support-parent/tooling-support/src/main/java/org/mule/runtime/module/tooling/internal/artifact/value/ValueProviderExecutor.java

```diff
@@ -91,12 +86,18 @@ public ValueResult resolveValues(ParameterizedModel parameterizedModel,
         return resultFrom(withContextClassLoader(extensionClassLoader, () -> valueProviderMediator.getValues(providerName,
                                                                                                              parameterValueResolver,
                                                                                                              connectionSupplier(context),
-                                                                                                             configSupplier(context))));
+                                                                                                             configSupplier(context)),
+                                                 ValueResolvingException.class,
```

Comment:
If you want, We have a utils method in org.mule.tooling.client.internal.utils.FunctionalUtils that wraps an exception from a Callable. So that the code is cleaner.

Reply:
That's on MTS side :(